docs/projects/version-control/converting/index.md

```diff
@@ -50,13 +66,28 @@ Once you press the **Configure** button, a modal window will appear to confirm t
 
 ![configuring version control](configure-version-control.png)
 
-Your project is now configured with Version Control. You can see this change reflected on the Process page, where you can change branches. You can also confirm this in your Git repository. The `.octopus` directory will now be created, and it should contain the following files:
+Your project is now configured with Version Control. You can see this change reflected on the left navigation of the page, where you can change branches. You can also confirm this in your Git repository. The `.octopus` directory will now be created, and it should contain the following files:
 
 - _deployment_process.ocl_
 - _deployment_settings.ocl_
 - _variables.ocl_
 - _schema_version.ocl_
 
+If your repository has branch protection setup, see [Setting up in a repository with protected branches](/docs/projects/version-control/converting/index.md#setting-up-in-a-repository-with-protected-branches).
+
+## Setting up in a repository with protected branches
+
+If your default branch is protected, you can select that option under Branch Settings. You will need to provide a different branch name for the initial commit. If the branch doesn't exist, it will be created. Once you click the **Configure** button, Octopus will commit the OCL file to the initial commit branch. 
+
+![initial commit branch and protected branches](version-control-configuration.png "width=500")
+
+Next, you will need to merge your changes into the default branch in your Git provide using your usual workflow. You will not be able to use the default branch within the project until you have merged your changes from the initial commit branch to the default branch. However, you can continue to make changes to the initial commit branch until then.
+
+Optionally, you can also nominate protected branches for your Project. This will prevent users from making direct commits to the nominated branches from the Octopus UI and encourage them to create a new branch instead. To nominate protected branches, type in the name or a wildcard pattern in the Protected Branches Pattern field under Branch Settings. This will apply to all existing and future branches.
+
+*Note that this is independant of your branch protection rules in your Git Provider and does not offer any protection outside of the Octopus UI.*
```

Comment:
```suggestion
If your repository has branch protection setup, see [Setting up in a repository with protected branches](/docs/projects/version-control/converting/index.md#setting-up-in-a-repository-with-protected-branches).

## Setting up a repository with protected branches

If your default branch is protected, you can select that option under Branch Settings. You will need to provide a different branch name for the initial commit. If the branch doesn't exist, it will be created. Once you click the **Configure** button, Octopus will commit the OCL file to the initial commit branch. 

![initial commit branch and protected branches](version-control-configuration.png "width=500")

Next, you will need to merge your changes into the default branch in your Git provider using your usual workflow. You will not be able to use the default branch within the project until you have merged your changes from the initial commit branch to the default branch. However, you can continue to make changes to the initial commit branch until then.

Optionally, you can also nominate protected branches for your Project. This will prevent users from committing directly to the nominated branches from the Octopus UI and encourage them to create a new branch instead. To nominate protected branches, type in the name or a wildcard pattern in the Protected Branches Pattern field under Branch Settings. This will apply to all existing and future branches.

*Note that this is independent of your branch protection rules in your Git Provider and does not offer any protection outside of the Octopus UI.*
```